lib/dns/signed_set.js

```diff
@@ -0,0 +1,9 @@
+class SignedSet{
```

Comment:
A 'singed set' is one that's been set on fire briefly. You want a 'signed set'.

Reply:
Fixed